0006-reward-ramp-for-packets.md

```diff
@@ -0,0 +1,105 @@
+- Start Date: May 21 2020
+- HIP PR: 
+- Tracking Issue: 
+
+# Summary
+[summary]: #summary
+
+Data Packets are already transferring on the network. Once the price oracle is in place, users will be able too burn HNT and generate Data Credits, use DC to send packets on the network, and Hotspots routing those packets will earn a reward in the form of HNT.
+This share is expected to be 30% of HNT per month, which is a large amount and can be easily gamed. 
+
+This proposal is to recommend a ramp up to 30% over a period of time to prevent undesirably gaming behaviour.
```

Comment:
What's the actual recommended ramp up and over what period of time? How does this conflict/work with the total ramp down of token distribution over the many year scale in the token distribution plan.